api/admin/routes.py

```diff
@@ -292,14 +329,12 @@ def admin_base(**kwargs):
 
 @app.route('/admin/static/circulation-web.js')
 @returns_problem_detail
-@requires_admin
```

Comment:
I think you took this out so that it could be accessed when the admin interface hasn't been set up yet. Is that right? Are there any other implications to making the JS and CSS public? (They seem public now.)